### PR TITLE
Added 100 licenses limit to teams-exploratory.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6264.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-exploratory.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Manage the Microsoft Teams Exploratory license
 
 The Microsoft Teams Exploratory experience lets users in your organization who have Azure Active Directory (Azure AD) and aren't licensed for Teams initiate an exploratory experience of Teams. Admins can switch this feature on or off for users in their organization. The earlier [Microsoft Commercial Cloud Trial](iw-trial-teams.md) is now replaced by The Teams Exploratory experience.
```
Comment (JohanFreelancer9) [inline Teams/teams-exploratory.md:26]:
```suggestion
The Microsoft Teams Exploratory experience lets users in your organization who have Azure Active Directory (Azure AD) and aren't licensed for Teams initiate an exploratory experience of Teams. Admins can switch this feature on or off for users in their organization. The earlier [Microsoft Commercial Cloud Trial](iw-trial-teams.md) is now replaced by the Teams Exploratory experience.
```

File: Teams/teams-exploratory.md [hunk 2]
```diff
@@ -125,7 +128,7 @@ To turn off the Teams Exploratory experience license:
 
 4. In the **Product licenses** pane, switch the toggle for this exploratory license to **Off**.
 
-    >[!Note]
+    >[!NOTE]
```
Comment (JohanFreelancer9) [inline Teams/teams-exploratory.md:131]:
```suggestion
    > [!NOTE]
```

File: Teams/teams-exploratory.md [hunk 3]
```diff
@@ -125,7 +128,7 @@ To turn off the Teams Exploratory experience license:
...
     >The Teams Exploratory toggle switch will appear after the first user in the organization launches the Teams Exploratory experience.
```
Comment (JohanFreelancer9) [inline Teams/teams-exploratory.md:132]:
```suggestion
    > The Teams Exploratory toggle switch will appear after the first user in the organization launches the Teams Exploratory experience.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor additional edits, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**